src/main/java/uk/gov/hmcts/ccd/domain/service/AccessControl.java

```diff
@@ -0,0 +1,4 @@
+package uk.gov.hmcts.ccd.domain.service;
+
+public interface AccessControl {
```

Comment:
do we really need this interface?

Reply:
@kiran-yenigala-hmcts see https://tools.hmcts.net/jira/browse/RDM-8860 for the rationale behind that interface
I think it's going to be very useful